src/components/common/DetailComment.vue

```diff
@@ -39,27 +37,36 @@ export default class DetailComment extends Vue {
     (this.$refs.textarea as any).resetValidation()
   }
 
-  /** Comment (v-model) passed into this component. */
+  /** Comment (v-model) passed into this component (required). */
   @Prop({ default: '' })
   private value: string
 
-  /** Label passed into this component. */
+  /** Placeholder passed into this component (optional). */
   @Prop({ default: '' })
-  private label: string
+  private placeholder: string
+
+  /** Max Length passed into this component (optional). */
+  @Prop({ default: 4096 })
+  private maxLength: number
 
-  /** Autofocus passed into this component. */
+  /** Autofocus passed into this component (optional). */
   @Prop({ default: false })
   private autofocus: boolean
 
+  /** Called when component is created. */
+  created (): void {
+    // inform parent of initial validity
+    this.emitValid(this.value)
+  }
+
   /**
    * Called when prop changes (ie, v-model is updated by parent).
    * This method is debounced to prevent excessive validation.
    */
   @Watch('value')
   @Debounce(300)
```

Comment:
I know this was not implemented in this PR, but this makes me think. Should we have implemented the Debounce decorator on all our listeners? Just a little bit of reading on the topic makes me think it is certainly not a bad thing when trying to optimize performance. No Action required here, just starting a conversation :)

Reply:
I implemented it here because it needed it (it validated a bunch of rules on every keystroke).

I think we should debounce when needed, and I haven't found any other candidates. Do you have something in mind?

Reply:
No nothing specific, i just know that some of the watchers are triggered constantly and was just thinking ( theoretically ) this is a cool way of optimizing. It's just something i'll keep in mind going forward, if a watcher is a good candidate for Debouncing.